resources/Components/RequestItem/index.js

```diff
@@ -37,7 +37,7 @@ class _RequestItem extends React.Component {
   }
   render() {
     const { account, handlerId, i, title, svgName, img, color, headerMode, txNonce, children } = this.props
-    const req = this.store('main.accounts', account, 'requests', handlerId)
+    const req = this.props.req || this.store('main.accounts', account, 'requests', handlerId)
```

Comment:
I think there's some inconsistency here, we should either be passing in the req or getting it from the store, not both and that could lead to a lot of confusion about how to use this component